packages/cryptoassets/src/currencies.ts

```diff
@@ -78,6 +78,25 @@ const ethereumUnits = (name, code) => [
 ];
 
 const cryptocurrenciesById: Record<string, CryptoCurrency> = {
+  near: {
+    type: "CryptoCurrency",
+    id: "near",
+    coinType: 0, //TODO: what should I put here?
+    name: "NEAR", //TODO: is it the name of the company, or name of the cryptocurrency?
+    managerAppName: "NEAR",
+    ticker: "N",
+    scheme: "near", //TODO: where this will be used? `near` is ok?
```

Comment:
typically it could allow (in future) a `near://` uri scheme. we could make this optional in future, you can leave it as "near"